starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -267,6 +267,7 @@ my %bad_gui;
 my %bad_id;
 my %config;
 my $db_handle;
+my $or_size;
```

Comment:
This global is not doing anything and should be removed. To revalidate just run one of the test cases and make sure it gives the same results.